include/client/libcurve.h

```diff
@@ -115,6 +115,20 @@ int IncreaseEpoch(const char* filename);
  */
 int Open(const char* filename, const C_UserInfo_t* userinfo);
 
+#define O_READONLY 0
+#define O_WRITE 1
+#define O_FAKEWRITE 2
```

Comment:
```suggestion
enum CurveOpenFlags {
    CURVE_EXCLUSIVE = 1 << 0,
    CURVE_SHARED = 1 << 1,
    CURVE_RDWR = 1 << 2,
    CURVE_RDONLY = 1 << 3,
    CURVE_WRONLY = 1 << 4,
    CURVE_FORCE_WRITE = 1 << 5,
};
```

might be better, `CURVE_WRONLY` may don't need, but other flags is necessary.